### ISSUE TITLE
"Validate" should be inactive as long as "Configuration Source" empty

### ISSUE THREAD

**Issue by Nokia808** [jkotra/eOVPN#10]
Hi.

I saw "Validate" is active even if "Configuration Source" is empty. So, I tried to click "Validate" while "Configuration Source" was still empty to see what it will happen. The "Validation" will go & continue for ever & can not stop unless by exit the application as a whole ! Even if you close "Setting" windows "Validation" will continue because when you re-open "Setting" you will see it already going !

This should be fixed by make "Validate" inactive as long as "Configuration Source" is still empty.

**Comment by jkotra** [comment]
Partially addressed in ver 0.23

Validate button is not visible by default. It is made visible when any of the three sources (url, ZIP or directory) is set. 

Part of settings window redesign.

**Comment by Nokia808** [comment]
@jkotra 
Hi. As I mentioned before, I'm not using VPN. But I tried to see what happened if I select any of 3 options (URL, ZIP, Directory).

1st of all, I kept the default selection (URL), & just tried to type "w". I noticed that "Validate" appearing. Then I tried to change selection to other options (ZIP & Directory) WITHOUT set any thing further (no selection for any ZIP specific file, nor selection for any specific directory), but I saw "Validate" still existing & active inspite that no thing really selected to be validated ! 

After that I returned to (URL) then tried to delete what I typed (just "w"), but "Validate remain appearing & active !

I think no think fixed or improved regarding "Validate" ..... No problem if it remain appearing. The problem is that it should be not active as long as no URL entered / no specific path selected ....
